FileManager/src/View/Helper/FileManagerHelper.php

```diff
@@ -183,7 +185,7 @@ public function linkDirectory($title, $path) {
  * @return string
  */
 	public function linkFile($title, $path) {
-		$output = "<a href='" . $this->Html->url(array('controller' => 'file_manager', 'action' => 'editfile')) . "?path=" . urlencode($path) . "'>{$title}</a>";
+		$output = "<a href='" . Router::url(array('controller' => 'Croogo/FileManager', 'action' => 'editFile')) . "?path=" . urlencode($path) . "'>{$title}</a>";
```

Comment:
I guess the controller is supposed to be 'FileManager' instead of 'Croogo/FileManager'. The same apply to the ones below